src/main/java/adris/altoclef/tasks/resources/wood/CollectSticksTask.java

```diff
@@ -0,0 +1,139 @@
+package adris.altoclef.tasks.resources.wood;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.Debug;
+import adris.altoclef.tasks.CraftInInventoryTask;
+import adris.altoclef.tasks.ResourceTask;
+import adris.altoclef.tasks.construction.DestroyBlockTask;
+import adris.altoclef.tasks.movement.PickupDroppedItemTask;
+import adris.altoclef.tasks.movement.TimeoutWanderTask;
+import adris.altoclef.tasksystem.Task;
+import adris.altoclef.util.CraftingRecipe;
+import adris.altoclef.util.ItemTarget;
+import adris.altoclef.util.RecipeTarget;
+import adris.altoclef.util.helpers.ItemHelper;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class CollectSticksTask extends ResourceTask {
+
+    private final int _targetCount;
+
+    private final Item[] _itemsToPickup;
+
+    public CollectSticksTask(int targetCount) {
+        super(Items.STICK, targetCount);
+        _targetCount = targetCount;
+        List<Item> itemsToPickup = new ArrayList<>(Arrays.asList(ItemHelper.LOG));
+        itemsToPickup.addAll(Arrays.asList(ItemHelper.PLANKS));
+        itemsToPickup.add(Items.DEAD_BUSH);
+        _itemsToPickup = itemsToPickup.toArray(new Item[itemsToPickup.size()]);
+    }
+
+    @Override
+    protected boolean shouldAvoidPickingUp(AltoClef mod) {
+        return false;
+    }
+
+    @Override
+    protected void onResourceStart(AltoClef mod) {
+        mod.getBehaviour().push();
+        mod.getBlockTracker().trackBlock(Blocks.DEAD_BUSH);
+        mod.getBlockTracker().trackBlock(ItemHelper.itemsToBlocks(ItemHelper.LOG));
+        mod.getBlockTracker().trackBlock(ItemHelper.itemsToBlocks(ItemHelper.PLANKS));
+    }
+
+    @Override
+    protected Task onResourceTick(AltoClef mod) {
+        // Calculate how much sticks can be made from planks and craft
```

Comment:
This all looks good to me, although I did this on my phone so I probably missed something. I will do a comprehensive review later.

I do have a few questions though. 

Do we really have to check the ground for loot? I'm pretty sure altoclef does that already.

Do we really need to re-define how we craft sticks? This is already a known recipie, so I get the feeling this is re-inventing the wheel. Really the only addition here is getting them from dead bushes. Is there any better place you can think of for that? 

Consider these questions, and let me know. I'm curious as to what you find.

Reply:
I do believe we have to check for grounded planks and logs manually since we don't have the crafting recipe for sticks in TaskCatalogue.java anymore (`// shapedRecipe2x2("stick", Items.STICK, 4, p, o, p, o);` is commented out), because you can't have a resource catalogued more than once. Therefore the bot can't know it needs logs and/or planks to craft sticks, that's why we need to both give him the recipe and tell him to pickup the wood. I have tried without those lines but the bot doesn't craft sticks in the first case or simply mines the block and then goes away without picking it in the second case (DestroyBlockTask doesn't try to get the items that dropped from the broken block).

Reply:
Picking up sticks is redundant as the resource task does that already. However this task also picks up logs, planks, and dead bushes. That works.

Reply:
We don't want to pickup daed bushes because we cannot craft with them. We only want to break them for sticks.